app/facades/merchants_facade.rb

```diff
@@ -0,0 +1,21 @@
+class MerchantsFacade
+  def initialize(id = nil)
+    @id = id
+  end
```

Comment:
No need to initialize this facade method. We dont need to store any state. Our facade is only responsible for calling to our service and then organizing that data. So, all of these methods that you call to in here should be class methods.